charts/netdata/values.yaml

```diff
@@ -148,6 +148,7 @@ parent:
       path: /etc/netdata/netdata.conf
       data: |
         [global]
+          hostname = netdata-parent
```

Comment:
Please update k8sState name too (it is Deployment, we should update its name too).